static/code/editor.js

```diff
@@ -440,15 +440,13 @@ Code.Editor.mostConfidentEditor = function() {
 /**
  * Set the values of all the editors.
  * @param {string} src Plain text contents.
- * @param {boolean} isSaved True if this is the saved source.
  */
-Code.Editor.setSourceToAllEditors = function(src, isSaved) {
+Code.Editor.setSourceToAllEditors = function(src) {
   Code.Editor.uncreatedEditorSource = src;
   for (var editor of Code.Editor.editors) {
     editor.setSource(src);
-    if (isSaved) {
-      editor.lastSavedSource = editor.getSource();
-    }
+    // Round-trip version of the source.
+    editor.lastSavedSource = editor.getSource();
```

Comment:
O.K., I think I understand how this works, but maybe `lastSavedSource` should be renamed?  It seems to me it's *not* the last saved source, but rather the just the unmodified-by-the-user source.

Reply:
Renamed to `unmodifiedSource`.